syntax = "proto2";

package proto;

message RobotInfo{
    required float radius = 1 [default = 0.09];
    required float height = 2 [default = 0.15];
    required float dribblerAngle = 3;
    required float frontWidth = 4;
}
